src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Options/RequestTimeoutOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Options;
+
+public class RequestTimeoutOptions
+{
+    public const string Section = "RequestTimeouts";
+
+    public int? RequestTimeoutMilliseconds { get; init; }
```

Comment:
Given we've opted to put this in a `RequestTimeout` section, we could simplify this key to just `TimeoutMilliseconds`.

I was also wondering if you could just put the default value (30000) in here as well? That way we don't need to put the default in `Startup`. Not entirely sure if this works as we'd expect or not though (would suggest checking).

Reply:
Renamed :)

I tested the default, but it doesn't work unfortunately :(